src/main/api/opds.ts

```diff
@@ -65,8 +65,13 @@ export class OpdsApi {
     }
 
     public async browse(data: any): Promise<THttpGetOpdsResultView> {
-        const { url } = data;
-        return await httpGet(url, {}, async (opdsFeedData) => {
+        let url: string = data.url;
+        if (new URL(url).protocol === "opds") {
```

Comment:
This code will always fail, because the `URL` `.protocol` accessor returns the `:` suffix. 

NodeJS API documentation: https://nodejs.org/api/url.html#url_url_protocol

Reduced repro test case: `node -e "console.log(new URL('https://domain.com').protocol)"` => `https:`

Reply:
Note that this is true for the legacy `URL` API as well:
`node -e "console.log(require('url').parse('https://domain.com').protocol)"`

Reply:
done https://github.com/readium/readium-desktop/pull/639/commits/0872fd8d706098f3a11553ef9c4f8184e8daa027